mllib/src/test/scala/org/apache/spark/ml/regression/LinearRegressionSuite.scala

```diff
@@ -998,6 +1047,172 @@ class LinearRegressionSuite
       }
     }
   }
+
+  test("linear regression (huber loss) with intercept without regularization") {
```

Comment:
Do you know if these integration tests are in the L1 penalty regime or in the L2 regime?  It'd be nice to make sure we're testing both.

Reply:
Yes, the test data was composed by two parts: inlierData and outlierData, and I have checked both regimes have been test. Thanks.